app/src/main/java/pm/gnosis/heimdall/ui/safe/add/AddSafeViewModel.kt

```diff
@@ -108,9 +111,10 @@ class AddSafeViewModel @Inject constructor(
 
     override fun addAdditionalOwner(input: String): Observable<Result<Unit>> =
             Observable.fromCallable {
-                val address = input.hexAsEthereumAddressOrNull()
```

Comment:
I would still disallow the direct input of something like `ethereum:0x0123...` and only use `parseEthereumAddress` for scanned results (in this case DeployNewSafeFragment). 

Anyways if you leave it in the view model can you add a test for this ;)